src/lib/consts.js

```diff
@@ -2,7 +2,7 @@ const os = require('os');
 const path = require('path');
 const { KEY_VALUE_STORE_KEYS } = require('@apify/consts');
 
-exports.DEFAULT_LOCAL_STORAGE_DIR = 'apify_storage';
+exports.DEFAULT_LOCAL_STORAGE_DIR = 'storage';
```

Comment:
This breaks current actors locally as it will use storage from `storage` and ignores `apify_storage`. What about:
* Remane folder apify_storage to storage during the first run, print warning we did it
* Mention in changelog that this folder name changed

Reply:
Makes sense. I wanted to mention it in release notes and I bumped the major version, but it's true that people would just install it anyway and then complain that it does not work 😄